bigframes/session/__init__.py

```diff
@@ -233,10 +233,14 @@ def read_gbq(
         index_col: Iterable[str] | str = (),
         col_order: Iterable[str] = (),
         max_results: Optional[int] = None,
+        columns: Iterable[str] = (),
+        filters: third_party_pandas_gbq.FiltersType = (),
         use_cache: bool = True,
         # Add a verify index argument that fails if the index is not unique.
     ) -> dataframe.DataFrame:
         # TODO(b/281571214): Generate prompt to show the progress of read_gbq.
+        query_or_table = self._filters_to_query(query_or_table, columns, filters)
```

Comment:
Please use `col_order` here. In a future PR, we can add `columns` as an alias for `col_order`. See: https://github.com/googleapis/python-bigquery-pandas/pull/701

```suggestion
        query_or_table = self._filters_to_query(query_or_table, col_order, filters)
```

Reply:
Done, changed in read_gbq.